aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/CustomConventions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+
+namespace WebAPIAnalyzers
+{
+public static class CustomConventions
+{
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesDefaultResponseType]
+    [ApiConventionNameMatch(ApiConventionNameMatchBehavior.Prefix)]
+    public static void Get(
+        [ApiConventionNameMatch(ApiConventionNameMatchBehavior.Suffix)]
+        int id)
+    { }
+}
+}
```

Comment:
Can you please fix the indentation in this class? You can see that our `CustomConventions` class is not indented properly when comparing it to the namespace.